### PR TITLE
Minor fixes across the codebase

### PR DESCRIPTION
**Pull request by alecalve**
- Updated some obsolete descriptions
- Moved functions from wallet.js to helpers.js
- Tighter checks for number validity (indexes must be positive integers, BTC amounts positive numbers, etc...)
- Removed some redundant variables

### CODE REVIEW

**Comment by Sjors** [comment]
@woudini can you check this for iOs. I'll do the same for web.

**Comment by pernas** [comment]
ack

File: src/api.js [hunk 1]
```diff
@@ -141,7 +141,7 @@ API.prototype.getTicker = function(){
 API.prototype.getUnspent = function(fromAddresses, confirmations){
   var data = {
       active : fromAddresses.join('|')
-    , confirmations : Helpers.isNumber(confirmations) ? confirmations : 0
+    , confirmations : Helpers.isPositiveNumber(confirmations) ? confirmations : 0
```
Comment (Sjors) [inline src/api.js]: This should be an integer (but do check).

File: src/blockchain-wallet.js [hunk 2]
```diff
@@ -108,8 +108,8 @@ Object.defineProperties(Wallet.prototype, {
     get: function() { return this._fee_per_kb;},
     set: function(value) {
       switch (true) {
-        case !Helpers.isNumber(value):
-          throw 'Error: wallet.fee_per_kb must be a number';
+        case !Helpers.isPositiveNumber(value):
```
Comment (Sjors) [inline src/blockchain-wallet.js]: Fee is in satoshi so should be an integer.

File: src/address.js [hunk 3]
```diff
@@ -69,30 +69,30 @@ Object.defineProperties(Address.prototype, {
     configurable: false,
     get: function() { return this._balance;},
     set: function(num) {
-      if(Helpers.isNumber(num))
+      if(Helpers.isPositiveNumber(num))
```
Comment (Sjors) [inline src/address.js]: I think these are all in satoshi, so can be integers.

**Comment by Sjors** [comment]
Retro ACK

**Pull request merged**